fastlane/Fastfile

```diff
@@ -89,17 +95,20 @@ platform :ios do
   desc "Release pods"
   private_lane :release_pods do
     AmplifyPods.pods.each do |pod|
-      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
+      if pod[:no_push]
+        next
+      end
 
-      UI.message("Pushing pod #{pod[:spec]}")
+      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
 
-      dupe_error = "[!] Unable to accept duplicate entry"
+      UI.message "Pushing pod #{pod[:spec]}"
 
-      begin
-        sh('bundle', 'exec', 'pod', 'trunk', 'push', pod_path, '--allow-warnings', '--synchronous')
-      rescue => ex
-        unless ex.to_s.include? dupe_error
-          raise Exception.new "Unrecoverable error in pod push trunk"
+      with_retry do
+        begin
+          sh('bundle', 'exec', 'pod', 'trunk', 'push', pod_path, '--allow-warnings', '--synchronous')
```

Comment:
As I read the code, the default retry period in `with_retry` is 5*60=300s=5 minutes; is that sufficient to catch CDN propagation? I recall that we had to wait up to 10 minutes in the past. Or is the `--synchronous` flag helping reduce that wait requirement?

Reply:
The purpose of this retry logic is solely to retry during intermittent failures, like the server error we experienced last week.

The `--synchronous` flag handles everything else, e.g., pushing to the trunk repo directly, instead of the CDN

Reply:
Thanks for the clarifications